### DISCUSSION TITLE
Using django-polymorphic with django-import-export

### DISCUSSION THREAD

**Discussion by ShaheedHaque** [jazzband/django-polymorphic#840] [Integrations]
I'm trying to use polymorphic models with the [Admin import-export](http://django-import-export.readthedocs.io/en/latest/api_admin.html#import_export.admin.ImportExportMixin) support provided by django-import-export. I've not (yet) looked into how this project does it's work, but a quick test reveals that the saved .csv file does not contain the fields from the subclass models (the saved data DOES contain the polymorphic_ctype as one might hope).

Now, I realise that the saved format of the data is a slightly tricky problem, especially if you have multiple levels of inheritance because the flat (relational-like!) format of a .csv file does not lend itself to the variable nature of the fields/columns. Maybe something like JSON encoding of the child fields might be needed?

At any rate, it would be nice to have this integration.

**Comment by bckohan** [comment]
Related issue on django-import-export: https://github.com/django-import-export/django-import-export/issues/1521

It seems like there were some admin incompatibilities that were fixed for the issue you filed.

@ShaheedHaque did you get it it work with django-polymorphic?
